tools/update-authors-json

```diff
@@ -86,14 +86,20 @@ def run_production() -> None:
             if contribs:
                 repos_done.append(name)
                 for contrib in contribs:
-                    if contrib.get('author') is None:
+                    assert contrib is not None  # TODO: To improve
+                    author = contrib.get('author')
+                    if author is None:
                         # This happens for users who've deleted their GitHub account.
                         continue
-                    username = contrib.get('author').get('login')
-                    contrib_data = {
-                        'avatar': contrib.get('author').get('avatar_url'),
-                        name: contrib.get('total'),
-                    }
+                    username = author.get('login')
+                    assert username is not None  # TODO: To improve
+                    try:
+                        contrib_data = {
+                            'avatar': author['avatar_url'],
+                             name: contrib['total'],
+                        }
+                    except KeyError:
+                        contrib_data = {'avatar': "", 'name': 0}  # TODO: To improve
```

Comment:
I'm not sure why this last thing is better than a `.get()` with a default?

Also, I think we actually just want to assert that `avatar_url` and `total` are present, probably?  Or is that not always true?

Reply:
Agreed about the `.get`, but also about the latter, in the simplest form which will pass mypy.